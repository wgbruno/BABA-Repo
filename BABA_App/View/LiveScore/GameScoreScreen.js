import React from "react";
import { View, Text, Image, StyleSheet, useWindowDimensions, ScrollView, Alert } from 'react-native';
import Logo from '../../assets/images/Logo_1.png'
import { NavigationContainer } from 'react-native';

export default function GameScoreScreen({navigation}){
  const {height} = useWindowDimensions();


  return (
      <ScrollView showsVerticalScrollIndicator={false}>
        <View style={styles.root}>
          <Image
            source={Logo}
            style={[styles.logo, {height: height * 20}]}
            resizeMode="contain"
          />
          </View>
      </ScrollView>
    );
};


const styles = StyleSheet.create({
    root: {
      alignItems: 'center',
      padding: 20,
    },
    logo: {
      width: '3000%',
      maxWidth: 132,
      maxHeight: 99,
    },
  });
  