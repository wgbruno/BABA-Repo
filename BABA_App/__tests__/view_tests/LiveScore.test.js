import {expect, jest, test} from '@jest/globals';
import renderer from 'react-test-renderer';
import GameScoreScreen from '../../View/LiveScore/GameScoreScreen';

test("Live Score Screen", function () {
    const ls_tree = renderer.create(<GameScoreScreen/>).toJSON();
    expect(ls_tree).toMatchSnapshot();
});