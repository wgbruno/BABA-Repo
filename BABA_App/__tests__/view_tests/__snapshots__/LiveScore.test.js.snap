// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Live Score Screen 1`] = `
<RCTScrollView
  showsVerticalScrollIndicator={false}
>
  <View>
    <View
      style={
        Object {
          "alignItems": "center",
          "padding": 20,
        }
      }
    >
      <Image
        resizeMode="contain"
        source={
          Object {
            "testUri": "../../../assets/images/Logo_1.png",
          }
        }
        style={
          Array [
            Object {
              "maxHeight": 150,
              "maxWidth": 200,
              "width": "3000%",
            },
            Object {
              "height": 26680,
            },
          ]
        }
      />
      <Text
        style={
          Object {
            "color": "darkblue",
            "fontSize": 25,
            "marginVertical": 10,
          }
        }
      >
                        
      </Text>
      <Text
        style={
          Object {
            "color": "gray",
            "fontSize": 40,
            "marginVertical": 10,
          }
        }
      >
                    
      </Text>
      <View
        accessible={true}
        collapsable={false}
        focusable={true}
        onBlur={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Array [
            Object {
              "alignItems": "center",
              "borderRadius": 5,
              "marginVertical": 5,
              "padding": 10,
              "width": "100%",
            },
            Object {
              "backgroundColor": "#DD4D44",
            },
            Object {},
          ]
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "white",
                "fontWeight": "bold",
              },
              undefined,
              Object {},
            ]
          }
        >
          Add 1 point to team 
        </Text>
      </View>
      <View
        accessible={true}
        collapsable={false}
        focusable={true}
        onBlur={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Array [
            Object {
              "alignItems": "center",
              "borderRadius": 5,
              "marginVertical": 5,
              "padding": 10,
              "width": "100%",
            },
            Object {
              "backgroundColor": "#DD4D44",
            },
            Object {},
          ]
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "white",
                "fontWeight": "bold",
              },
              undefined,
              Object {},
            ]
          }
        >
          Remove 1 point from team 
        </Text>
      </View>
      <View
        accessible={true}
        collapsable={false}
        focusable={true}
        onBlur={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Array [
            Object {
              "alignItems": "center",
              "borderRadius": 5,
              "marginVertical": 5,
              "padding": 10,
              "width": "100%",
            },
            Object {
              "backgroundColor": "#DD4D44",
            },
            Object {},
          ]
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "white",
                "fontWeight": "bold",
              },
              undefined,
              Object {},
            ]
          }
        >
          Add 1 point to team 
        </Text>
      </View>
      <View
        accessible={true}
        collapsable={false}
        focusable={true}
        onBlur={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Array [
            Object {
              "alignItems": "center",
              "borderRadius": 5,
              "marginVertical": 5,
              "padding": 10,
              "width": "100%",
            },
            Object {
              "backgroundColor": "#DD4D44",
            },
            Object {},
          ]
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "white",
                "fontWeight": "bold",
              },
              undefined,
              Object {},
            ]
          }
        >
          Remove 1 point from team 
        </Text>
      </View>
      <View
        accessible={true}
        collapsable={false}
        focusable={true}
        onBlur={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onResponderGrant={[Function]}
        onResponderMove={[Function]}
        onResponderRelease={[Function]}
        onResponderTerminate={[Function]}
        onResponderTerminationRequest={[Function]}
        onStartShouldSetResponder={[Function]}
        style={
          Array [
            Object {
              "alignItems": "center",
              "borderRadius": 5,
              "marginVertical": 5,
              "padding": 10,
              "width": "100%",
            },
            Object {
              "backgroundColor": "#DD4D44",
            },
            Object {},
          ]
        }
      >
        <Text
          style={
            Array [
              Object {
                "color": "white",
                "fontWeight": "bold",
              },
              undefined,
              Object {},
            ]
          }
        >
          Return Home
        </Text>
      </View>
    </View>
  </View>
</RCTScrollView>
`;
